test/crypto/eax.js

```diff
@@ -142,6 +143,11 @@ module.exports = () => describe('Symmetric AES-EAX', function() {
   };
 
   describe('Symmetric AES-EAX (native)', function() {
+    before(function () {
+      const detectNative = () => !!(util.getWebCrypto() || util.getNodeCrypto());
+      detectNative() || this.skip();
```

Comment:
I think this would be simpler to read as just
```suggestion
      if (!util.getWebCrypto() && !util.getNodeCrypto()) {
        this.skip();
      }
```